ui/components/app/assets/token-list/token-list.tsx

```diff
@@ -36,18 +37,18 @@ function TokenList({ onTokenClick }: TokenListProps) {
     chainIds: chainIdsToPoll as Hex[],
   });
 
-  const nonEvmNativeToken = useNativeTokenBalance();
+  const multichainAssets = useMultiChainAssets();
 
   // network filter to determine which tokens to show in list
   // on EVM we want to filter based on network filter controls, on non-evm we only want tokens from that chain identifier
   const { networkFilter } = useNetworkFilter();
 
   const sortedFilteredTokens = useMemo(() => {
-    const balances = isEvm ? evmBalances : [nonEvmNativeToken];
+    const balances = isEvm ? evmBalances : multichainAssets;
     const filteredAssets: TokenWithFiatAmount[] = filterAssets(balances, [
       {
         key: 'chainId',
-        opts: isEvm ? networkFilter : { [nonEvmNativeToken.chainId]: true },
+        opts: isEvm ? networkFilter : { solana: true },
```

Comment:
Would this break Bitcoin?

Reply:
Yeah it would. This should be the `assetId.namespace`

Will update